src/libraries/pkg/descriptions.json

```diff
@@ -1024,7 +1024,7 @@
     ]
   },
   {
-    "Name": "System.IO.Compression.clrcompression",
+    "Name": "System.IO.Compression.Native",
```

Comment:
I would either leave it alone, or delete it.

Reply:
I will delete this then.